lib/resources/StagedSalesforceData.ts

```diff
@@ -43,7 +43,7 @@ export class StagedSalesforceDataResource extends Resource<
         return this.makeRequest({
             method: 'PUT',
             data: updateArgs,
-            url: `${this.basePath}/{id}`,
```

Comment:
So the interpolation occurs inside of `makeRequest` you don't need to do this here. What is the issue you're trying to resolve?